framework/src/org/checkerframework/framework/util/MultiGraphQualifierHierarchy.java

```diff
@@ -717,6 +717,10 @@ assert getTopAnnotation(a1) == getTopAnnotation(a2) :
         }
         return newglbs;
     }
+    
+    protected boolean annotationMirrorsAreEqual(AnnotationMirror a1, AnnotationMirror a2) {
```

Comment:
No problem.  I think in the future it will save time if I review all the code changes at once.